standup_notes/__main__.py

```diff
@@ -158,11 +145,16 @@ def delete_notes(amount):
                 break
             else:
                 print("Please enter a valid response")
-
-
     else:
         print("No files to be deleted")
 
 
+def validate(date_text):
+    try:
+        datetime.datetime.strptime(date_text, '%Y-%m-%d')
```

Comment:
You could remove the `import datetime` line and add datetime to the existing datetime imports (from datetime import date, _datetime_, ...) and then change this to just `datetime.strptime`.

Reply:
Just pushed the code up